articles/android-studio-shimmer-loading-effect/index.md

```diff
@@ -533,7 +546,7 @@ public class MainActivity extends AppCompatActivity {
 }
 ```
 
-As you can see above, the code implements loading data from the internet using the Volley library into the RecyclerView. Here we load the data in the JSON URL we included above. We load the data and display it into the main activity. In this case, we load the RecyclerView content into the main screen of our application.
+As you can see above, the code implements loading data from the Internet using the Volley library into the RecyclerView. Here we load the data in the JSON URL we included above. We load the data and display it into the main activity. In this case, we load the RecyclerView content into the main screen of our application.
```

Comment:
Could you some synonyms for the word load -- loading --loaded. It seems to be overused